File: ID008/csharp/Program.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        var inputs = Console.ReadLine().Split(" ");
        var n = int.Parse(inputs[0]);
        var s = int.Parse(inputs[1]);
        var ans = 0;
        for (int i = 1; i <= n; i++)
        {
            for (int j = 1; j <= n; j++)
            {
                if (i + j <= s) ans++;
            }
        }
        Console.WriteLine(ans);
    }
}

File: ID008/dart/program.dart
import 'dart:io';

void main(List<String> args) {
  var inputs = stdin.readLineSync()?.split(" ").map(int.parse).toList() ?? [];
  var n = inputs[0];
  var s = inputs[1];
  var ans = 0;
  for (var i = 1; i <= n; i++) {
    for (var j = 1; j <= n; j++) {
      if (i + j <= s) ans++;
    }
  }
  print(ans);
}
